src/components/NumberInput/NumberInput.scss

```diff
@@ -0,0 +1,23 @@
+@use '../../styles/scss/typography.scss';
+@use '../../styles/scss/paddings.scss';
+@use '../../styles/scss/margins.scss';
+@use '../../styles/scss/colors.scss';
+@use '../../styles/scss/mixins.scss';
+@use '../../styles/scss/variables.module.scss';
+
+.#{variables.$prefix}-number-input {
+  padding: paddings.$xs;
+  margin: margins.$xs;
+
+  color: colors.$text;
+
+  @include typography.text-regular();
+
+  input {
+    width: 20%;
```

Comment:
For a super simple responsive input I recommend setting:
- `width` to 100% so that the input takes up all of the available space (the consumer will decide how much that is)
- a `min-with` the input should not become narrower then, so that on to small containers it will force a scrollbar, instead of allowing the contents to overflow
- a `max-width the input should not become narrower then, so that on big screens it does not take up an unreasonably amount of space